proxy/failing_filter_test.go

```diff
@@ -0,0 +1,292 @@
+package proxy_test
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	stdlibhttptest "net/http/httptest"
+	"net/url"
+	"sync/atomic"
+	"testing"
+	"time"
+
+	"github.com/zalando/skipper/eskip"
+	"github.com/zalando/skipper/filters"
+	"github.com/zalando/skipper/filters/builtin"
+	"github.com/zalando/skipper/filters/shedder"
+	"github.com/zalando/skipper/net/httptest"
+	"github.com/zalando/skipper/proxy"
+	"github.com/zalando/skipper/proxy/proxytest"
+)
+
+func TestResponseFilterOnProxyError(t *testing.T) {
+	counter := int64(1)
+	serverErrN := int64(37)
+	timeoutN := int64(5)
+	timeout := 100 * time.Millisecond
+
+	backend := stdlibhttptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		atomic.AddInt64(&counter, 1)
+
+		v := atomic.LoadInt64(&counter)
+		if v%timeoutN == 0 {
+			time.Sleep(timeout) // 499 no response
+			return
+		} else if v%serverErrN == 0 {
+			w.WriteHeader(500)
+			w.Write([]byte("FAIL"))
+		} else {
+			w.WriteHeader(200)
+			w.Write([]byte("OK"))
+		}
+	}))
+	defer backend.Close()
+
+	var routes = fmt.Sprintf(`
+                main: * -> admissionControl("mygroup", "active", "1s", 5, 0, 0.99, 0.9, 1.0) -> "%s";
+        `, backend.URL)
+	r := eskip.MustParse(routes)
+
+	fr := make(filters.Registry)
+	fr.Register(shedder.NewAdmissionControl(shedder.Options{}))
```

Comment:
I think we need to create and configure postProcessor for it.

Reply:
Yes true that might help

Reply:
I tested it and it does not help. PostProcessor is only working on routing updates.